### ISSUE TITLE
[Achievement] Mountain o' Mounts

### ISSUE THREAD

**Issue by Aveta** [Valhalla-Gaming/Tracker#4191]
The reward for this achievment is sent to my mail every time I log on, making for a lot of junk in my mail if I don't delete it.

**Comment by Collector-Valhalla** [comment]
Indeed. I'm keep getting that mail with mount on each of my characters everytime I log in.
